src/adr/0019-polyglot-strategy.md

```diff
@@ -1,6 +1,6 @@
 # 19. Polyglot codebase
```

Comment:
You gonna need to change the number of the ADR (in the file too), but can be done during the merge